app/templates/reports/reports.html

```diff
@@ -70,6 +70,7 @@ <h2 class="ui header">
                             <th>Vehicle ID</th>
                             <th>License Plate</th>
                             <th>Duration</th>
+                            <th>Weather conditions during incident</th>
```

Comment:
oh, my comment was to add the word "condition". i don't have very strong feelings about this, but i think being explicit is better. if it looks terrible then i'm ok with shortening it, but otherwise i don't see the harm in being explicit.